setup.py

```diff
@@ -457,6 +457,9 @@ def run(self):
     directives['linetrace'] = True
     macros = [('CYTHON_TRACE', '1'), ('CYTHON_TRACE_NOGIL', '1')]
 
+# in numpy>=1.16.0, silence build warnings about deprecated API usage
+macros.append(('NPY_NO_DEPRECATED_API', '0'))
```

Comment:
> These are warnings we can't do anything about; they are an artifact of cython+numpy interaction.

This would be a useful comment to include to explain why we're doing this.